components/backup-stream/src/metrics.rs

```diff
@@ -155,9 +156,11 @@ lazy_static! {
         &["stage"]
     )
     .unwrap();
-    pub static ref LOST_LEADER_REGION: IntCounter = register_int_counter!(
```

Comment:
For reviewers: this series hasn't been uploaded into any of the official Grafana dashboards, it is probably safe to rename it.